config/database.yml.sqlite

```diff
@@ -0,0 +1,11 @@
+development:
```

Comment:
Ah, i think this has to be removed -- see this for some help with that!

https://stackoverflow.com/questions/215718/reset-or-revert-a-specific-file-to-a-specific-revision-using-git

👍

Reply:
Looking great - can you try removing this file from the PR? Thanks!

Reply:
Ah, still this one file to be removed!